components/dashboard/src/workspaces/workspace-model.ts

```diff
@@ -136,7 +136,7 @@ export class WorkspaceModel implements Disposable, Partial<GitpodClient> {
         return (
             info.workspace.pinned ||
             (!!info.latestInstance && info.latestInstance.status?.phase !== 'stopped')
-        ) && !info.workspace.softDeleted;
+        );
```

Comment:
**Nit**: Why do you consider deleted/garbage-collected workspaces as active?

Reply:
I'm not, but I'll revert anyways. 
Looking at the workspace-db-impl.ts it inconsistently checking for `softDeleted`.

Reply:
reverting...